fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -263,8 +287,32 @@ public Builder useLegacyScrubber(boolean legacy) {
                 return this;
             }
 
+            /**
+             * Choose a specific "Already Scrubbed" rangeSet prefix. This may be useful for
+             * multiple scrubbing jobs that may not affect each other.
+             * 0 is the backward compatible default, which means no prefix.
+             * @param rangeId a rangeSet prefix
+             * @return this builder
+             */
+            public Builder setRangeId(final int rangeId) {
+                this.rangeId = rangeId;
+                return this;
+            }
+
+            /**
+             * If set to true, clear any previous "Already Scrubbed" range set (in the given rangeId) - which means that
+             * the index scrubbing will begin from scratch.
```

Comment:
I feel like the second part of this is much clearer, so perhaps this would be better:
```suggestion
             * Start the scrubbing from scratch, regardless of any progress already made for this scrubber id.
```